### ISSUE TITLE
SSL certificate error when downloading compiler with version 1.2

### ISSUE THREAD

**Issue by atifaziz** [atifaziz/LinqPadless#51]
This is a follow-on issue reported in #49. It seems the patch applied in PR #50 was only a partial fix. It works as long as a LINQPad Query file contains a reference to at least one NuGet package.

### Steps to Reproduce

1. Install LINQPadless 1.2: `nuget install LinqPadless -Version 1.2.0`
2. Create a LINQPad Query file called `Test.linq` with the following content (note that the blank line is significant):  
    ```
    <Query Kind="Expression" />
    
    DateTime.Now
    ```
3. Compile an executable: `LinqPadless.1.2.0\tools\lpless.exe -t exe Test.linq`

### Expected Behaviour

Expected the compilation to succeed and generate an executable.

### Actual Behaviour

    A:\tmp\Test.linq
    Packages directory: A:\tmp\packages
    Packages target: .NETFramework,Version=v4.5.2
    Installing C# compiler using NuGet:
        Downloading NuGet from https://dist.nuget.org/win-x86-commandline/v3.5.0/nuget.exe
    The request was aborted: Could not create SSL/TLS secure channel.

### Workaround

Reference at least one NuGet package. For example, changing `Test.linq` from above with the following content instead:

```
<Query Kind="Expression">
  <NuGetReference Version="3.0.0">Mannex</NuGetReference>
</Query>

DateTime.Now
```

produces an executable without an error when `LinqPadless.1.2.0\tools\lpless.exe -t exe Test.linq` is retried.

---
/cc @veremenko-y

**Comment by atifaziz** [comment]
Closed via e44ae301eaa8175df0d2bfff24ea2eed520a0016 that is the merge of PR #52.

**Comment by atifaziz** [comment]
The fix is now published with release [1.2.1](https://github.com/atifaziz/LinqPadless/releases/tag/v1.2.1).
